x-pack/plugins/monitoring/public/directives/beats/beat/index.js

```diff
@@ -21,9 +22,11 @@ uiModule.directive('monitoringBeatsBeat', (timefilter) => {
 
       function onBrush({ xaxis }) {
         scope.$evalAsync(() => {
```

Comment:
No need for `scope.$evalAsync(` here or elsewhere it is wrapping calls to `timefilter`.